fun main() {
    try {
        error("")
    } finally {
        println("err")
    }
}